src/picknik_ur_base_config/objectives/cartesian_sequence.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="Cartesian Sequence">
+  <BehaviorTree ID="Cartesian Sequence" _description="Plan and execute a Cartesian path between a sequence of poses" _favorite="true">
+    <Control ID="Sequence" name="root">
+      <!-- Create demo waypoints -->
+      <!-- Waypoint 1 -->
+      <Action ID="CreateStampedPose" reference_frame="manual_grasp_link" position_xyz="-0.1;0.0;0.0" orientation_xyzw="0.0;0.0;0.0;1.0" stamped_pose="{stamped_pose}"/>
+      <Action ID="AddPoseStampedToVector" input_pose="{stamped_pose}" pose_stamped_vector="{pose_stamped_vector}"/>
+      <!-- Waypoint 2 -->
+      <Action ID="CreateStampedPose" reference_frame="manual_grasp_link" position_xyz="-0.1;0.0;0.1" orientation_xyzw="0.0;0.0;0.0;1.0" stamped_pose="{stamped_pose}"/>
+      <Action ID="AddPoseStampedToVector" input_pose="{stamped_pose}" pose_stamped_vector="{pose_stamped_vector}"/>
+      <!-- Waypoint 3 -->
+      <Action ID="CreateStampedPose" reference_frame="manual_grasp_link" position_xyz="0.0;0.0;0.1" orientation_xyzw="0.0;0.0;0.0;1.0" stamped_pose="{stamped_pose}"/>
+      <Action ID="AddPoseStampedToVector" input_pose="{stamped_pose}" pose_stamped_vector="{pose_stamped_vector}"/>
+      <!-- Plan Cartesian motion -->
+      <Action ID="InitializeMTCTask" task_id="cartesian_sequence" controller_names="/joint_trajectory_controller /robotiq_gripper_controller" task="{cartesian_sequence_task}"/>
```

Comment:
This is done in a lot of tasks for some reason but why do we need the `/robotiq_gripper_controller` here?